Cookbook.ipynb

```diff
@@ -360,107 +331,104 @@
      ]
     }
    ],
    "source": [
-    "from multiversx_sdk_core.transaction_builders import ESDTTransferBuilder\n",
+    "from multiversx_sdk_core import Token, TokenTransfer\n",
     "\n",
-    "payment = TokenPayment.fungible_from_amount(\"TEST-8b028f\", \"100.00\", 2)\n",
+    "token = Token(\"TEST-8b028f\")\n",
+    "transfer = TokenTransfer(token, 10000)\n",
     "\n",
-    "builder = ESDTTransferBuilder(\n",
-    "    config=config,\n",
+    "transaction = transfer_factory.create_transaction_for_esdt_token_transfer(\n",
     "    sender=alice,\n",
     "    receiver=bob,\n",
-    "    payment=payment\n",
+    "    token_transfers=[transfer]\n",
     ")\n",
     "\n",
-    "tx = builder.build()\n",
-    "print(\"Transaction:\", tx.to_dictionary())\n",
-    "print(\"Transaction data:\", tx.data)"
+    "print(\"Transaction:\", transaction.__dict__)\n",
+    "print(\"Transaction data:\", transaction.data)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Create a single NFT transfer:"
+    "Create a single NFT transfer:\n",
+    "\n",
+    "Keep in mind, since we are sending a NFT, we `should` set the amount to `1`."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
+   "execution_count": 13,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Transaction: OrderedDict([('nonce', 0), ('value', '0'), ('receiver', 'erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th'), ('sender', 'erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th'), ('gasPrice', 1000000000), ('gasLimit', 1213500), ('data', 'RVNEVE5GVFRyYW5zZmVyQDU0NDU1MzU0MmQzMzM4NjYzMjM0MzlAMDFAMDFAODA0OWQ2MzllNWE2OTgwZDFjZDIzOTJhYmNjZTQxMDI5Y2RhNzRhMTU2MzUyM2EyMDJmMDk2NDFjYzI2MThmOA=='), ('chainID', 'D'), ('version', 2), ('signature', '')])\n",
-      "Transaction data: ESDTNFTTransfer@544553542d333866323439@01@01@8049d639e5a6980d1cd2392abcce41029cda74a1563523a202f09641cc2618f8\n"
+      "Transaction: {'chain_id': 'D', 'sender': 'erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th', 'receiver': 'erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th', 'gas_limit': 1213500, 'nonce': 0, 'amount': 0, 'data': b'ESDTNFTTransfer@544553542d333866323439@01@01@8049d639e5a6980d1cd2392abcce41029cda74a1563523a202f09641cc2618f8', 'signature': b'', 'sender_username': '', 'receiver_username': '', 'gas_price': 1000000000, 'version': 2, 'options': 0, 'guardian': '', 'guardian_signature': b''}\n",
+      "Transaction data: b'ESDTNFTTransfer@544553542d333866323439@01@01@8049d639e5a6980d1cd2392abcce41029cda74a1563523a202f09641cc2618f8'\n"
      ]
     }
    ],
    "source": [
-    "from multiversx_sdk_core.transaction_builders import ESDTNFTTransferBuilder\n",
-    "\n",
-    "payment = TokenPayment.non_fungible(\"TEST-38f249\", 1)\n",
+    "token = Token(\"TEST-38f249\", 1)\n",
+    "transfer = TokenTransfer(token, 1)\n",
```

Comment:
set named parameter for amount here? so it is more clear that the comment from above is related to token amount?